### PR TITLE
Refactored namespaces

### PR DESCRIPTION
**Pull request by aprokopenko**

### CODE REVIEW

File: src/Boilerplates.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	 */
+	public static function theme( Event $event ) {
+		$io = $event->getIO();
+
+		$theme_name = str_replace( 't=', '', $event->getArguments()[1] );
+		$theme_slug = str_replace( ' ', '', $theme_name );
+		$theme_dir  = str_replace( ' ', '', $event->getArguments()[0] );
+		if ( empty( $theme_dir ) ) {
+			$theme_dir = 'wp-content/themes';
+		}
```
Comment (aprokopenko) [inline src/Boilerplates.php]: wrong parameters getter.

folder/slug - is a trim(1st parameter without keys)
title  - you need to find argument with -t
namespace - you need to find argument with -ns
destination dir    - you need to find argument with -f

it's better to make helper, which can parse event arguments and provide array like:

['key' => value, ... (for options), 0 => value, 1 => value (for params without options)]

File: src/Boilerplates.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+		if ( is_dir( $src ) ) {
+			$dir = opendir( $src );
+			if ( ! is_dir( $dst ) ) {
+				if ( ! mkdir( $dst, 0777, true ) ) {
+					$io->write( "\n ERROR. Unable to create directory $theme_dir" );
+
+					return false;
+				}
+			}
```
Comment (aprokopenko) [inline src/Boilerplates.php]: very weird if case. you check for is_dir, and then check for !is_dir how this can be?

File: src/Boilerplates.php [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+		}
+
+	}
+
+
+	/**
+	 * @param: source folder $src
+	 * @param: destination folder $dst
+	 */
+	public static function recurse_copy( $src, $dst ) {
```
Comment (aprokopenko) [inline src/Boilerplates.php]: this method will be used in plugin installation as well.
I suggest to make class inside src/Helpers/FileSystem (the same as init wp command)
and use it

File: src/Boilerplates.php [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+			$theme_dir = 'wp-content/themes';
+		}
+
+		$src  = 'vendor/justcoded/wordpress-theme-boilerplate';
+		$dst  = $theme_dir . '/' . $theme_slug;
+		self::$args = array(
+			'full_path_to_new_theme' => $dst,
+			'theme_slug'             => $theme_slug,
+			'theme_name'             => $theme_name,
+		);
```
Comment (aprokopenko) [inline src/Boilerplates.php]: using static vars is not good way to pass variables through code. this is the same as using global. this is wrong

File: src/Boilerplates.php [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+//			$file = file_get_contents( $filename );
+//			file_put_contents(
+//				$filename,
+//				preg_replace(
+//					"/(Boilerplate)/",
+//					ucfirst( $theme_slug ), $file
+//				)
+//			);
+//		}
+//	}
```
Comment (aprokopenko) [inline src/Boilerplates.php]: clean unnecessary code

**Comment by aprokopenko** [comment]
@mzhirov please update

File: src/Boilerplates.php (hunk 1 above)
Comment (mzhirov) [reply src/Boilerplates.php]: ready

File: src/Boilerplates.php (hunk 4 above)
Comment (mzhirov) [reply src/Boilerplates.php]: done

File: src/Boilerplates.php (hunk 2 above)
Comment (mzhirov) [reply src/Boilerplates.php]: done

File: src/Boilerplates.php (hunk 3 above)
Comment (mzhirov) [reply src/Boilerplates.php]: done

File: src/Boilerplates.php (hunk 5 above)
Comment (mzhirov) [reply src/Boilerplates.php]: done

**Pull request merged**